### ISSUE TITLE
Training Data of  xlm-roberta-large-finetuned-conll03-* models

### ISSUE THREAD

**Issue by wangxinyu0922** [huggingface/transformers#6474]
Hi, I'm curious about the training data of xlm-r models finetuned on conll ner datasets (e.g.  xlm-roberta-large-finetuned-conll03-german,  xlm-roberta-large-finetuned-conll03-english), are the models trained on train+dev sets?

**Comment by LysandreJik** [comment]
Pinging @stefan-it

**Comment by stefan-it** [comment]
Hi @wangxinyu0922 ,

the models are only trained on the corresponding training data sets, that means development data was not used for training :)

**Comment by wangxinyu0922** [comment]
That's great! Thank you!

**Comment by wangxinyu0922** [comment]
> 
> 
> Hi @wangxinyu0922 ,
> 
> the models are only trained on the corresponding training data sets, that means development data was not used for training :)

By the way, what is the accuracy of the model on the four datasets? The models are trained on document context or sentence context? I believe different context will affect the performance.
